driver/driver_test.go

```diff
@@ -31,13 +31,16 @@ import (
 )
 
 var netAdapter string
+var vswitchName string
 var controllerAddr string
 var controllerPort int
 var useActualController bool
 
 func init() {
 	flag.StringVar(&netAdapter, "netAdapter", "Ethernet0",
 		"Network adapter to connect HNS switch to")
+	flag.String("vswitchName", "Layered Ethernet0",
```

Comment:
In May 2017 update of Windows Server vswitch names created alongside HNS networks are named `Layered_<ifname>`.